src/vivarium_nih_us_cvd/components/healthcare_visits.py

```diff
@@ -241,9 +345,143 @@ def visit_doctor(
 
         return df
 
-    def update_treatment(self, index: pd.Index):
-        # TODO [MIC-3371, MIC-3375]
-        pass
+    def update_treatment(self, df: pd.DataFrame) -> Tuple[pd.DataFrame, pd.Index]:
```

Comment:
This function appears to be called only in one place, so extracting it to a new function only serves to make the code harder to follow.